server/plugins/inputs/index.ts

```diff
@@ -60,6 +60,7 @@ const builtInInputs = [
 ];
 
 for (const input of builtInInputs) {
+	// @ts-ignore
```

Comment:
Same thing here, needs a comment as to why it's safe to ignore or a fix of the root cause